base-notebook/Dockerfile

```diff
@@ -171,6 +171,11 @@ RUN sed -re "s/c.ServerApp/c.NotebookApp/g" \
     /etc/jupyter/jupyter_server_config.py > /etc/jupyter/jupyter_notebook_config.py && \
     fix-permissions /etc/jupyter/
 
+# This healtcheck works well for `lab`, `notebook`, `nbclassic`, `server` and `retro` jupyter commands
```

Comment:
Thank you @consideRatio!
These are very good suggestions and I have a lot to learn from you :+1:

I'm a little bit afraid you're spending a lot of time reviewing my PRs, hope it's ok for you (because one more is coming soon).